Formula/gradle.rb

```diff
@@ -1,14 +1,14 @@
 class Gradle < Formula
   desc "Build system based on the Groovy language"
   homepage "https://www.gradle.org/"
-  url "https://downloads.gradle.org/distributions/gradle-3.4.1-all.zip"
+  url "https://downloads.gradle.org/distributions/gradle-3.5-all.zip"
   sha256 "ed7e9c8bb41bd10d4c9339c95b2f8b122f5bf13188bd90504a26e0f00b123b0d"
 
   bottle :unneeded
 
   option "with-all", "Installs Javadoc, examples, and source in addition to the binaries"
 
-  depends_on :java => "1.7+"
+  depends_on :java => "1.8+"
```

Comment:
Thanks for your contribution, @MatheusRV. Where'd you get 1.8+ from? https://gradle.org/install still says 1.7+. Also, there already is #12299, which you might have noticed had you paid attention to the PR template.